app/views/pages/about.html.erb

```diff
@@ -11,16 +11,16 @@
 <main>
   <h1>About</h1>
 
-  <section class="empowering-women">
-    <h2 class="section-title">Empowering Women with Ruby</h2>
+  <section class="empowering-through-ruby">
+    <h2 class="section-title">Empowering through Ruby</h2>
     <p>Do you dream of someday writing software that helps people and improves the world?</p>
-    <p>Led by an all-volunteer team of seasoned, enthusiastic Ruby and Rails developers, the workshop introduces women of all backgrounds to the concepts, tools, and techniques of Ruby and Rails development. Our audience is those with no or little programming experience.</p>
+    <p>Led by an all-volunteer team of seasoned, enthusiastic Ruby and Rails developers, the workshop introduces people of all backgrounds to the concepts, tools, and techniques of Ruby and Rails development. Our audience is those with no or little programming experience.</p>
     <p>We welcome you to the Boston Ruby community. Whatever your goal is in learning to program, we hope to connect you with the tools to take the next step.</p>
   </section>
 
   <section class="promoting-diversity">
     <h2 class="section-title">Promoting Diversity</h2>
-    <p>The Boston software scene needs more women. So this workshop is for women and their guests. You should identify as a woman, or be the male guest of a woman who is attending.</p>
+    <p>The Boston software scene needs more people with diverse backgrounds, so this workshop is for those who are in underrepresented groups in tech. These include, but are not limited to, those who identify as women, people of color, LGBTQ, military veterans, and more. If you do not fall under these or similar groups, you can still join us as someone's guest.</p>
```

Comment:
I like that. 

Are you suggesting we omit the part about guests @edwardloveall? I would prefer to keep it, maybe reworded. 

I like that you & @mxie managed to phase that without repeating 'underrepresented in tech' several times.